.github/workflows/tests.yml

```diff
@@ -41,6 +41,9 @@ jobs:
       # however, it still gets the modified files and we can continue
       # Error: The head commit for this pull_request event is not ahead of the base commit.
       continue-on-error: true
+    # save workspaces to speed up testing
+    - name: set CAPA_SAVE_WORKSPACE environment variable
+      run: echo "CAPA_SAVE_WORKSPACE=True" >> $GITHUB_ENV
```

Comment:
i think can maybe use a top level element to set the env var for all steps, like:

```yaml
env:
    CAPA_SAVE_WORKSPACE: "True"
```

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#env

Reply:
nice find, updated in 441550d